stable/database/README.md

```diff
@@ -413,21 +389,12 @@ $ nuocmd delete database --db-name demo
 $ nuocmd show domain
 ```
 
-Then you must also clear the PVCs:
```

Comment:
The PVCs are not automatically cleaned, aren't they?

Reply:
They aren't, and neither are the PV's...

Reply:
PVCs are automatically cleared out when you delete the statefulset. The PVs are deleted based on the reclaim policy. This line was incorrect.

Reply:
Yes our reclaim is retain...

Reply:
I don't see this behavior - the PVC is always retained and the PV deleted based on reclaim policy happens when the PVC is deleted, not the statefulset.  I just tried again in OCP, deleting with Helm and via the UI...  In Rancher I've always had to clear down the PVC's and PV's after deleting workloads.

If this wasn't the case, the PV wouldn't get re-used on redeployment of the same statefulset (e.g if you delete the PVC but not the PV, re-deploy would create new PVC and PV)

Reply:
Ok. I will add it back.